src/pages/Explorer/Layouts/SheetsTemplate.svelte

```diff
@@ -1,25 +1,43 @@
 <script>
   import Svg from 'webkit/ui/Svg/svelte'
+  import { dialogs } from 'webkit/ui/Dialog'
+  import { userSubscription } from '../store'
+  import UpgradeModal from '../Components/UpgradeModal.svelte'
 
-  export let item
+  export let item = {}
 
   let isOpen = false
+
+  $: ({ description, name, url } = item)
+  $: isPro = $userSubscription.isPro || false
 </script>
 
 <div on:click={() => (isOpen = !isOpen)}>
   <div class="row justify v-center">
-    <h5>{item.title}</h5>
+    <h5>{name}</h5>
     <Svg id="arrow-down" w="10" height="5.5" class="$style.arrow {isOpen && '$style.open'}" />
   </div>
   {#if isOpen}
     <div class="mrg-l mrg--t mrg--b description">
       <p class="mrg-s mrg--b">
-        {item.desc}
+        {description}
       </p>
-      <a href={item.url} target="_blank" class="btn-1 btn--s row v-center">
-        <div class="mrg-s mrg--r">Open template</div>
-        <Svg id="external-link" w="12" />
-      </a>
+      {#if isPro}
+        <a href={url} target="_blank" class="btn-1 btn--s row v-center">
+          <div class="mrg-s mrg--r">Open template</div>
+          <Svg id="external-link" w="12" />
+        </a>
+      {:else}
+        <!-- svelte-ignore a11y-invalid-attribute -->
+        <a
+          href="#"
+          class="btn-1 btn--s row v-center"
+          on:click|preventDefault={() => dialogs.show(UpgradeModal)}
```

Comment:
Export `dialogs.show(UpgradeModal)` as the `showUpdateModal` function from the `Components/UpgradeModal.svelte` module so it can be reused `src/pages/Explorer/Layouts/WeeklyReport.svelte` as well